src/renderer/util/theme.js

```diff
@@ -103,6 +103,23 @@ export const addThemeStyle = theme => {
   }
 }
 
+export const setEditorWidth = value => {
+  const EDITOR_WIDTH_STYLE_ID = 'editor-width'
+  let result = ''
```

Comment:
You should provide the default editor width here. Otherwise if `value` is incorrect the maximal width is used (?).

```css
`:root { --editorAreaWidth: 750px; }`
```

Reply:
I think no need to provide default value? Because the default value is already in `theme` css files, we just add an empty `style` tag if the `editorLineWidth ` is invalid. but we still use the   `--editorAreaWidth` in theme css files.